blobfuse/blobfuse.cpp

```diff
@@ -68,6 +70,10 @@ const struct fuse_opt option_spec[] =
     OPTION("--pre-mount-validate=%s", pre_mount_validate),
     OPTION("--background-download=%s", background_download),
     OPTION("--invalidate-on-sync=%s", invalidate_on_sync),
+    OPTION("--streaming=%s", streaming),
+    OPTION("--stream-cache-mb=%s", stream_buffer),
+    OPTION("--max-block-per-file=%s", max_block_per_file),
+    OPTION("--block-size-mb=%s", block_size_mb),
```

Comment:
yes I have not planned to do that, smaller block size means more calls converging to container and higher network latency. To optimize the performance side just assumed it will always be in MBs only.